### DISCUSSION TITLE
setting 13.6 Demodu

### DISCUSSION THREAD

**Discussion by VintageEngineer** [egzumer/uv-k5-firmware-custom#460] [Q&A] [answered]
setting 13.6 Demodu    
The default setting seems to be AM, which surprised me? Maybe I tweaked that by mistake. Is the default FM?
Is it possible to configure this from chirp? I cannot find it in the Settings there...

**Comment by egzumer** [comment, marked as answer]
AM is not default.
You can set that in chirp, it is called `Mode` there.

**Reply by VintageEngineer** [reply]
Thank you! I see it is at the end of the Memories section there are settings for each band. yay!
Awesome.
